### DISCUSSION TITLE
Traduction

### DISCUSSION THREAD

**Discussion by MattiaPARRINELLO** [upptime/upptime#315] [Q&A] [answered]
Hi, 
There is a way to traduct the page ?

**Comment by AnandChowdhary** [comment]
What does `traduct` mean?

**Reply by MattiaPARRINELLO** [reply]
Translate to an other language

**Comment by AnandChowdhary** [comment, marked as answer]
Yes, translations are supported: https://upptime.js.org/docs/configuration#internationalization
